man/fi_msg.3.md

```diff
@@ -173,6 +173,11 @@ to write CQ entries for all successful completions.  See the flags
 discussion below for more details. The requested message size that
 can be used with fi_inject is limited by inject_size.
 
+If FI_HMEM is enabled, the fi_inject call can only accept buffer with
+iface equal to FI_HMEM_SYSTEM if the provider requires FI_MR_VIRT_ADDR
```

Comment:
Did you mean FI_MR_HMEM instead of FI_MR_VIRT_ADDR?

Reply:
Yes. Blame the co-pilot ...